src/components/MessageBar/MessageBar.scss

```diff
@@ -27,13 +30,18 @@ $MessageBar-padding: 8px;
 }
 
 .ms-MessageBar-icon {
-  padding-right: $MessageBar-padding;
+  @include padding-right($MessageBar-padding);
   font-size: $ms-icon-size-m;
   @include ms-fontColor-neutralSecondaryAlt;
+  min-width: 16px;
+  min-height: 16px;
+  display:flex;
```

Comment:
Also, I'm seeing a lot of display: flex but not a lot of additional flex properties. Are those props really needed?